--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -144,15 +144,7 @@ if(NOT asio_location OR asio_location ST
 endif()
 add_definitions(-DASIO_STANDALONE)
 
-if (APPLE OR WIN32 OR
-    CMAKE_SYSTEM_NAME MATCHES "FreeBSD" OR
-    CMAKE_SYSTEM_NAME MATCHES "NetBSD" OR
-    CMAKE_SYSTEM_NAME MATCHES "OpenBSD")
-  if (NOT CMAKE_SYSTEM_NAME MATCHES "kFreeBSD")
-    find_package(Intl REQUIRED)
-  endif()
-endif()
-
+find_package(Intl)
 # Disable no symbols warning on macOS, but only on versions where this is
 # supported. It is not supported on OS X 10.7 (DARWIN_MAJOR_VERSION == 11).
 if (APPLE)
--- a/cmake/WlFunctions.cmake
+++ b/cmake/WlFunctions.cmake
@@ -185,7 +185,7 @@ macro(_common_compile_tasks)
     # libintl is not used on all systems, so only include it, when we actually
     # found it.
     if (Intl_FOUND)
-      wl_include_system_directories(${NAME} ${Intl_INCLUDE_DIRS})
+      wl_include_system_directories(${NAME} ${MINIZIP_INCLUDE_DIRS} ${Intl_INCLUDE_DIRS})
       target_link_libraries(${NAME} ${TARGET_LINK_FLAGS} ${Intl_LIBRARIES})
     endif()
   endif()
