--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -328,7 +328,7 @@ if(NOT MSVC)
          OR (NOT (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 13.0)) AND (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 13.1)))
     # Skip this on g++ versions 12.0, 12.1, and 13.0 because of false-positives from system headers.
     message(WARNING "This compiler is known to cause false-positive warnings.")
-  else()
+  elseif(FALSE)
     # Turn some warnings into errors.
     message(STATUS "Warnings will be treated as errors.")
     wl_add_flag(WL_COMPILE_DIAGNOSTICS "-Werror")
