--- a/src/base/macros.h
+++ b/src/base/macros.h
@@ -128,22 +128,6 @@
 #define PRINTF_FORMAT(b, c)
 #endif
 
-#ifdef PRIuPTR
-#define PRIuS PRIuPTR
-#else
-#ifdef _WIN32
-#ifdef _WIN64
-#define PRIuS PRIu64
-#else
-#define PRIuS PRIu32
-#endif
-#else
-#if SIZE_MAX == UINT64_MAX
-#define PRIuS "lu"
-#else
-#define PRIuS "u"
-#endif
-#endif
-#endif
+#define PRIuS "zu"
 
 #endif  // end of include guard: WL_BASE_MACROS_H
