--- a/ccsm.appdata.xml.in
+++ b/ccsm.appdata.xml.in
@@ -4,12 +4,12 @@
   <id type="desktop">ccsm.desktop</id>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-2.0+</project_license>
-  <_name>CCSM</_name>
-  <_summary>Settings Manager for Compiz (CCSM)</_summary>
+  <name>CCSM</name>
+  <summary>Settings Manager for Compiz (CCSM)</summary>
   <description>
-    <_p>
+    <p>
       Compiz Config and Settings tool (CCSM).
-    </_p>
+    </p>
   </description>
   <url type="homepage">https://gitlab.com/compiz/ccsm</url>
 </component>
--- a/ccsm.desktop.in
+++ b/ccsm.desktop.in
@@ -1,6 +1,6 @@
 [Desktop Entry]
-_Name=CompizConfig Settings Manager
-_Comment=Configure Compiz with CompizConfig
+Name=CompizConfig Settings Manager
+Comment=Configure Compiz with CompizConfig
 Icon=ccsm
 Exec=ccsm
 Terminal=false
--- a/setup.py
+++ b/setup.py
@@ -130,11 +130,13 @@ f = open (os.path.join ("ccsm"), "wt")
 f.write (data)
 f.close ()
 
-cmd = "intltool-merge -d -u po/ ccsm.desktop.in ccsm.desktop".split(" ")
+cmd = ["msgfmt", "--desktop", "--template", "ccsm.desktop.in",
+       "-d", "po", "-o" "ccsm.desktop"]
 proc = subprocess.Popen(cmd)
 proc.wait()
 
-cmd = "intltool-merge -x -u po/ ccsm.appdata.xml.in ccsm.appdata.xml".split(" ")
+cmd = ["msgfmt", "--xml", "--template", "ccsm.appdata.xml.in",
+       "-d", "po", "-o" "ccsm.appdata.xml"]
 proc = subprocess.Popen(cmd)
 proc.wait()
 
