--- a/Makefile.am
+++ b/Makefile.am
@@ -49,15 +49,15 @@ conf_DATA += profiles/network/network.co
 state_DATA =
 endif
 
+dbussystembusdir = $(DBUS_SYSTEMBUSDIR)
+dbussystembus_DATA = src/org.bluez.service
+
 if SYSTEMD
 systemdsystemunitdir = $(SYSTEMD_SYSTEMUNITDIR)
 systemdsystemunit_DATA = src/bluetooth.service
 
 systemduserunitdir = $(SYSTEMD_USERUNITDIR)
 systemduserunit_DATA =
-
-dbussystembusdir = $(DBUS_SYSTEMBUSDIR)
-dbussystembus_DATA = src/org.bluez.service
 endif
 
 EXTRA_DIST += src/org.bluez.service
